### PR TITLE
zstandard 0.22.0

### PR DESCRIPTION
**Pull request by cbouss**
For PKG-4513 authenticode signing

- Update to 0.22.0
- use system zstd (see comments in recipe).

https://github.com/indygreg/python-zstandard/tree/0.22.0
https://github.com/indygreg/python-zstandard/releases

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,30 +1,41 @@
...
-  sha256: 31d12fcd942dd8dbf52ca5f6b1bbe287f44e5d551a081a983ff3ea2082867863
+  sha256: 8226a33c542bcb54cd6bd0a366067b610b41713b64c9abec1bc4533d69f51e70
+  patches:
+    - use_system_zstd.patch
 
 build:
   number: 0
-  skip: True  # [py<36]
+  skip: True  # [py<38]
+  script: {{PYTHON}} -m pip install --no-deps --no-build-isolation . -vv
```
Comment (danpetry) [inline recipe/meta.yaml:18]:
```suggestion
  script: {{ PYTHON }} -m pip install --no-deps --no-build-isolation . -vv
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,30 +1,41 @@
...
+    - patch     # [unix]
+    - m2-patch  # [win]
   host:
     - python
-    - cffi 1.15
+    - cffi 1.16
     - pip
     - setuptools
     - wheel
+    - zstd {{zstd_version}}
```
Comment (danpetry) [inline recipe/meta.yaml:31]:
```suggestion
    - zstd {{ zstd_version }}
```

**Review by danpetry — APPROVED** [review]
couple of nits

**Review by SC426 — APPROVED** [review]
LGTM once Dan's comments are addressed

**Pull request merged**